modules/rgbd/src/dynafu.cpp

```diff
@@ -153,7 +153,7 @@ DynaFuImpl<T>::DynaFuImpl(const Params &_params) :
                           params.raycast_step_factor)),
     pyrPoints(), pyrNormals(), warpfield()
 {
-#ifdef HAVE_OPENGL
+#ifdef HAVE_GLEW
```

Comment:
Will this work on Windows?

Reply:
I'm not sure. I don't have Windows at my disposal. Can you check if the PR works for you?

Reply:
Checked, it does not work on Windows. Maybe because HAVE_GLEW is not a proper way to check if GLEW is enabled (I don't really know how to check it, maybe through GLEW_Found or similar thing?)

Reply:
@savuor Check `cvconfig.h` changes in corresponding PRs into the main repository.

Reply:
The current code version works on Windows without any 3rdparty dependencies. The change you want to merge makes user to install glew. Can this be avoided? Or we need glew for some other functionality?